src/components/MyListItem/MyListItem.css

```diff
@@ -8,6 +13,36 @@ td.item {
   border-color: #000;
 }
 
+td.item .button-div {
+  width: 100%;
+  height: 100%;
+  white-space: nowrap;
+  display: flex;
+  flex-direction: row;
+}
+
+td.item .button-div button {
```

Comment:
specificity makes the CSS hard to mantain 

(if you ever tried to change the style and it was not changing, that is the case)

the easy solution is to actually give a class name to that button, it can be something like `activate-button`

Reply:
you can read more here: 
https://css-tricks.com/strategies-keeping-css-specificity-low/